src/main/java/net/neoforged/neoforge/registries/DeferredHolder.java

```diff
@@ -0,0 +1,263 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.registries;
+
+import com.mojang.datafixers.util.Either;
+import java.util.Locale;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+import net.minecraft.core.Holder;
+import net.minecraft.core.HolderOwner;
+import net.minecraft.core.Registry;
+import net.minecraft.core.registries.BuiltInRegistries;
+import net.minecraft.resources.ResourceKey;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.tags.TagKey;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * A Deferred Holder is a Holder that is constructed with only a ResourceKey.<br> It will be populated with the underlying Holder from the registry when available.
```

Comment:
Please don't use hard breaks. Either have it continue in the same paragraph, or split it into a separate paragraph.

Apply this same suggestion to other uses of `<br>` in this class, and others.

Reply:
All the `<br>` are gone.